ddtrace/tracer/sqlcomment.go

```diff
@@ -73,9 +74,11 @@ func (c *SQLCommentCarrier) Inject(spanCtx ddtrace.SpanContext) error {
 		if traceID == 0 {
 			traceID = c.SpanID
 		}
-		tags[sqlCommentTraceID] = strconv.FormatUint(traceID, 10)
-		tags[sqlCommentSpanID] = strconv.FormatUint(c.SpanID, 10)
-		tags[sqlCommentKeySamplingPriority] = strconv.Itoa(samplingPriority)
+		sampled := int64(0)
+		if samplingPriority > 0 {
+			sampled = 1
```

Comment:
hypothetically other things could cause this to be sampled (like if there ends up being an error it might get picked up by the error sampler even though it was priority 0) But from reading the spec it appears fine for us to just use a "Best effort" sampling priority when passing the info along

Reply:
I think that's right. And, for what it's worth, we've discussed including `tracestate` once there's more clarity and definition around tracestate support in tracers. That's currently being discussed but once that's available, I think we'd likely get the full sampling decision bits propagated. For now, having the `sampled` is fine for our database monitoring use-cases.